notebooks/mc-wave-routing-demo.ipynb

```diff
@@ -0,0 +1,333 @@
6Fuj7G7g5jB/M/DzMH8G0d33Ao4C3g/lvYHi8NorzPdqhtiOBw4FPkpFbMC0sK3Cvqc3c6w/Am6sZdsx4d+7EzAsfA8y6/tOAE8DF4X5PwLf3oNYBwKHhvluwMIQU6v7bOuJtdV9tuFY88J8NvB++AxqrR/4DvDHMH8R8FRTj6EZY30EOL+W7Vv0O+BnHHVrcKytNDoH+HOY/zPR/SzV5X+xyFSgp6L7Xk4DXjGz9Wa2AXiFaPDIPWJmbwHrUxFbWNfdzKZa9C3/S6yu5oq1LucAE82szMwWA0VE34davxPhL7WTiQbqrHncTYl1lZl9EOa3APOJhutpdZ9tPbHWJW2fbfh8tobF7DBZPfXHP+9ngC+GeBp1DM0ca11a9DvgiaNujR1rK1UMeFnSDEkTQtkA23UH/WfAgDBfV8wteSzNFdugMF+zvLldF07tH6pu+mlCrH2AjWZW2dyxhuaRQ4j+4mzVn22NWKEVfraSMiXNIho49RWiM4S66t8ZU1i/KcTTIv/PasZqZtWfa21j9bXod8ATR+v3BTM7lGh4+mslHR9fGf5aaJXXVLfm2II/ACOAccAq4P+lN5zdScoDngX+y8w2x9e1ts+2llhb5WdrZlVmNo5oCKQjgP3SHFKdasYq6UCisfr2Aw4nan66KR2xeeKoW1JjbaWama0Ir2uIhmE5AlitXUOvDCT6iwTqjrklj6W5YlsR5lMWs5mtDv85E8CDRJ9tU2JdR9Q0kFWjvMkkZRP9ED9uZn8Lxa3ys60t1tb82Yb4NgJvAEfXU//OmML6HiGeFv1/Fot1fGgaNIvG9XuYpn+ue/YdSLYzpKNNRHfVFxN1flV3dB3QwjHkAt1i8+8S9U3cw+6dpHeH+TPZvYNsmu3qIFtM1DnWK8z3bqYY89m9w7nZYuPznXdnNHOsA2Pz/4eo3RrgAHbv/Cwm6vis8zsB/JXdO1i/swdxiqjN+Vc1ylvdZ1tPrK3usyV6pk/PMN8F+DdwVl31A9eye+f40009hmaMdWDsc/8V8LN0fAda7EewLU5EVyosJGoHvTUN7z88fPk+BOZWx0DUzvoa8AnwauyLIKKnLi4C5hA9XbG6rm8QdeIVAV9vpvieJGqGqCBqI72qOWMjGmr/o7DP7wgjHTRjrI+GWGYTDcAZ/7G7NbzvAmJXm9T1nQj/VtPCMfwV6LQHsX6BqBlqNtF4bLPC+7a6z7aeWFvdZwscRPTU0dnh2G+vr36gc1guCuuHN/UYmjHW18Pn+hHRsEzVV1616HfAhxxxzjnXKN7H4ZxzrlE8cTjnnGsUTxzOOecaxROHc865RvHE4ZxzrlE8cbhWSVJVbATQWdUjjUp6U1JBGuJ5RNL5NcruC7HNk7Q9Fuv5kl6U1DMFcQyU9EJz19tUikbBvbGe9Wcpely0a0dS+uhY5/bAdouGW2i1zOxa2DlG0ws14n2mtn2awQ1Ed2K3FZOBn0j6mZmVpjsY1zz8jMO1WZL+IKkw/ryCUL5E0t3hWQPTJI0M5f8h6aPwjIO3QlmmpHskTQ8Dx10dyiXpd+HZCq8C/RsZ2xJJfSXlS/o4nLEslPS4pFMkvROej3BE2D43DAY4TdJMSXWNqnoe8K+wzwFh+1kh9lGh/LJY+f2SMkP5eEkfhON/LZT1lvR82H+qpINC+Y9CPG9KKpZ0fezYbg3H8jawb6z8+nD2NVvSRNg5ptabRHc9u/aiqXc2+uRTKiegil13Is8CLgzlbxLuimXXndOZofygsLyEXXfZX050NgDRHbWDwnz1cA4TgNvCfCegkGjIiK8SjZ6aCewNbKSW5yCE/fKJDWUSi6FvWFcJjCX6Q20G8BDRnb7nAM+H7X8KXFYdG9Hdx7k16hwGzIgt/xa4NMznEA1NsT/wDyA7lP8+fAb9iEZJHVbjs/st8MMwfzIwK8z/iGiIm07hONYRDe19WPgcuwLdie5GvjHss5Jdd133jMV5KfDbdH+nfGq+yZuqXGuVTFPVBYqGms8ieqDQGKIhGiAaYqT69Zdh/h3gEUlPA9UDB34JOCjWf9EDGEX04KcnzawKWCnp9T04lsVmNgdA0lzgNTMzSXOIEkt1HGfH+gs6A0OJnm9RbSBQElt+D7hV0mDgb2b2iaQvEv24T1f0QLcuRIMhHgW8ZdHzIzCz6meTfIHoLAYze11SH0ndw7rJFg2mVyZpDdEw7scBz1lodpI0KRbPbOBxSc8Dz8fK1xAlX9dOeOJwbZKkYcCNwOFmtkHSI0Q/ttWs5ryZXSPpSKIB4WZIOozoL///NLOXatR/Bs2nLDafiC0n2PV/UMB5Zragnnq2EztGM3tC0vtEx/NiaGYT8GczuyW+o6Qv72HcVTT8e3EmUcL9MlFCG2vRcyw6h9hdO+F9HK6t6g5sAzZJGkD0vJK4C2Ov7wFIGmFm75vZ7UR/uQ8BXgK+HYYGR9JoSbnAW8CFoQ9kIHBSio/nJeA/FU4TJB1SyzYL2XWGgqThQLGZ/Qb4O9HAeK8B50vqH7bpLWkfYCpwfEi4SOodqvk3UVMSkk4E1lqNZ3/U8BZwrqQukroRJQkkZQBDzOwNomdE9ADywj6jiQbTc+2En3G41qqLoqefVfuXmd1cvWBmH0qaCXxM1Hb/To39e0maTfRX88Wh7J7QgSyiH9gPiZpX8oEPwo92CdEjNJ8javOfBywlJJ8U+gnRMNmzw4/wYmp0KJvZNkmLJI00syLgAuBrkiqIngj4UzNbL+k2oqdGZhCNBnytmU0NzXp/C+VrgFOJ+jIeCp9VKXBFfUGa2QeSniL67NYA08OqTOAxST2IPt/fWPQcCYiS7i2fq8y1WT46rmt3JC0h6kBfm+5YmpukrwCHmdlt6Y4lGeFs8Akz+2K6Y3HNx884nGtDzOw5SX3SHUcjDAW+l+4gXPPyMw7nnHON4p3jzjnnGsUTh3POuUbxxOGcc65RPHE455xrFE8czjnnGuX/AwDnii5/KLS9AAAAAElFTkSuQmCC\n",
```

Comment:
Usually, we'd want to store this without the image information embedded in the notebook. @awlostowski-noaa Let's talk about what makes most sense.

Reply:
Noted. I am going to get into the practice of saving notebooks after I have cleared all outputs.